incubator/nifi/values.yaml

```diff
@@ -0,0 +1,214 @@
+  persistence:
+    enabled: true
+    ## A manually managed Persistent Volume and Claim
+    ## Requires persistence.enabled: true
+    ## If defined, PVC must be created manually before volume will be bound
+    # existingClaim:
+
+    ## NiFi data Persistent Volume Storage Class
+    ## If defined, storageClassName: <storageClass>
+    ## If set to "-", storageClassName: "", which disables dynamic provisioning
+    ## If undefined (the default) or set to null, no storageClassName spec is
+    ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
+    ##   GKE, AWS & OpenStack)
+    ##
+    # storageClass: "-"
+    annotations: {}
+    accessMode: ReadWriteOnce
+    size: 8Gi
+    volumes:
+    #  - name: nothing
+    #    emptyDir: {}
+    mounts:
+    #  - mountPath: /var/nothing
+    #    name: nothing
+    #    readOnly: true
+
+    ## Storage Capacities for persistent volumes
+    # Storage capacity for the 'data' directory, which is used to hold things such as the flow.xml.gz, configuration, state, etc.
+    dataStorage: 1Gi
+    # Storage capacity for the FlowFile repository
+    flowfileRepoStorage: 10Gi
+    # Storage capacity for the Content repository
+    contentRepoStorage: 10Gi
+    # Storage capacity for the Provenance repository. When changing this, one should also change the properties.provenanceStorage value above, also.
+    provenanceRepoStorage: 10Gi
+    # Storage capacity for nifi logs
+    logStorage: 5Gi
+
+## Configure resource requests and limits
+## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #  cpu: 100m
+  #  memory: 128Mi
+  # requests:
+  #  cpu: 100m
+  #  memory: 128Mi
+
+## Node labels for pod assignment
+## Ref: https://kubernetes.io/docs/user-guide/node-selection/
+nodeSelector: {}
+
+## Tolerations for pod assignment
+## Ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+tolerations: []
+
+# Affinity for pod assignment
+# Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+affinity: {}
+
+################################################################################
+## Zookeeper
+################################################################################
+
+zookeeper:
+  ## Zookeeper chart
+  ## ref: https://github.com/kubernetes/charts/tree/master/incubator/zookeeper
+  enabled: true
+
+  # Number of ZooKeeper replicas
+  servers: 1
+
+  ## Configure Zookeeper resource requests and limits
+  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  resources:
+    limits:
```

Comment:
Resource limits should be commented out. We expect this to be opt-in so folks can run this in varying environments.